spec/features/campaign_overview_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe 'campaign overview page', type: :feature, js: true do
+  let(:slug)  { 'spring_2016' }
+  let(:user) { create(:user) }
+  let(:admin) { create(:admin) }
+  let(:campaign) {
+    create(:campaign,
+           id: 10001,
+           title: 'My awesome Spring 2016 campaign',
+           slug: slug,
+           description: 'This is the best campaign')
+  }
+
+  before do
+    Capybara.current_driver = :poltergeist
+    stub_token_request
+  end
+
+  context 'as an user' do
+    it 'should not show the edit buttons' do
+      login_as(user, scope: user)
+      visit "/campaigns/#{campaign.slug}"
+      sleep 1
+      expect(page).to have_no_css('.campaign-description .editable-edit')
+    end
+  end
+
+  context 'as an admin' do
+    before do
+      login_as(admin, scope: :user)
+      visit "/campaigns/#{campaign.slug}"
+      sleep 1
```

Comment:
These pass locally for me without the sleep. Did you find this necessary, or was it just following the pattern in other specs?